src/witan/models/dem/ccm/schemas.clj

```diff
@@ -62,3 +62,11 @@
 (def AtRiskPopnSchema
   (make-ordered-ds-schema [[:gss-code s/Str] [:sex s/Str] [:age s/Int] [:year s/Int]
                            [:popn s/Num] [:actualyear s/Int] [:actualage s/Int]]))
+
+;; For the migration component
+(def MigrationEstimatesSchema
```

Comment:
this schema is going to be applicable for all the components coming from the MYEB2_CoC.csv. I would suggest changing to ComponentMYESchema